src/mesh/boundary_info.C

```diff
@@ -2601,5 +2622,76 @@ void BoundaryInfo::_find_id_maps(const std::set<boundary_id_type> & requested_bo
   // to save memory, also ought to reserve memory
 }
 
+void BoundaryInfo::clear_stitched_boundary_side_ids (const boundary_id_type sideset_id,
+                                                     const boundary_id_type other_sideset_id,
+                                                     const bool clear_nodeset_data)
+{
+  auto end_it = _boundary_side_id.end();
+  auto it = _boundary_side_id.begin();
+
+  auto predicate =
```

Comment:
Maybe a comment about what this `predicate()` lambda does? Either here or when it's used. It is sufficiently complicated looking to warrant that IMO.

Reply:
comment added in https://github.com/libMesh/libmesh/pull/2850/commits/5a32f242b27eb4a62092d22ede56992b98f8cac3